CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 ## Unreleased
-### Fixes
 - Shell escape pane titles to fix multi word and special character titles
+- Add configuration options for pane titles
```

Comment:
Would you mind rebasing with `master` to fix up the changelog?

Reply:
Done